velero-2-mixin/targets.libsonnet

```diff
@@ -100,5 +100,115 @@ local utils = commonlib.utils {
       )
       + prometheusQuery.withLegendFormat('%s - failure' % utils.labelsToPanelLegend(this.config.legendLabels)),
 
+    lastBackupStatus:
+      prometheusQuery.new(
+        '${' + vars.datasources.prometheus.name + '}',
+        'velero_backup_last_status{%(queriesSelector)s}' % vars
```

Comment:
The queries for this dashboard still have the extra hardcoded job value appended to the selectors, did you learn anything about those in terms of what the standard is? Otherwise I say they can be removed.